### PR TITLE
[Chore: #163460156] Setup Staging App

### PR DESCRIPTION
**Pull request by ebenezerdon**
#### What does this PR do?
This PR adds details for the new hosted applications in `travis.yml'

#### Description of Task to be completed?
Setup staging app

#### How should this be manually tested?
This PR should enable automatic deploy from Travis CI

#### What are the relevant pivotal tracker stories?
[#163460156](https://www.pivotaltracker.com/story/show/163460156)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -18,13 +18,29 @@ before_deploy:
...
+    run: npm run db:migrate
+    app: authors-haaven
+    on:
+      repo: andela/pippin-ah-backend
+      branch: develop
+  - provider: heroku
+    api_key:
+      secure: CgZwp0km3S2QcsGUT67oBBajcbPw03Kw2Prd2K4/hTlr/VSZQoIn3WeMXu2fIB0aaeVWZtcaA8tPhzt8bNOCXUDnROInxx4tMvjSIZlsXoK7rcqXOi5j4sj6B5VnXtpJtocqC+Aa8kRUrOmNVkpBLvY20uvwOdJxeNEF14jZPCADD/Yh8UC214gp4sZ+2OSeAc3GQgn/VhVfoCo6hT3yiBGrNTUmW/ti+YSDw+WZE3G55+ZTj2pvPAv68mrEQR9HTSXnpoqDj9gWt6ZObDu8ysrxu5sWCxMACDfhoay+kcGl4B3ga2GMwW3J3z44XbtQ66946EMmf99/0XpznKc1r4abAfAzmv2Jcevk/SfnAH62KCvw02WHgnpjPsrps/p2DhDhl55YtlJO55wTfm1tGYPXoIXHvT5t3MmvDhc+igdQ2u/WFAOEg3d7prf7cAmb1pqnTWQhUKC36Q5lrKb/GCSmWK9+lg8NLBZ65ifjPVfXzryQZavl39MZUVpurC7Hjm2gEWapuD8rKFJeNJ+Dq9w8UcykX9BKKu+39mSEm3NK1w/8j50TyAapRt5C9uPUzgW8xeMUqKQCyLyV6iyku1ahgh/1QSCgTLouabS6QihVh8pd5R4fZVy/hpzhNTaUG98WzRSMir1FWYpmeHOjhgIkrv8bD34DddTn1KSZS8E="
```
Comment (oreoluwade) [inline .travis.yml:22]: @ebenezerdon Just to clarify, we'd be deploying 3 apps now, yes?

Comment (ebenezerdon) [reply .travis.yml:22]: It's supposed to be two. Can I remove the initial one?

**Comment by oreoluwade** [comment]
@ebenezerdon could you check the conflict?

I had removed the artifacts bit in the travis config earlier. Please remove it in this branch too.

**Comment by ebenezerdon** [comment]
I've done so

**Pull request merged**